packages/alchemy/src/client/multisigAccountClient.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  accountLoupeActions,
+  createMultisigModularAccount,
+  multisigPluginActions,
+  multisigSignatureMiddleware,
+  pluginManagerActions,
+  type AccountLoupeActions,
+  type CreateMultisigModularAccountParams,
+  type LightAccount,
+  type MultisigModularAccount,
+  type MultisigPluginActions,
+  type MultisigUserOperationContext,
+  type PluginManagerActions,
+} from "@alchemy/aa-accounts";
+import {
+  smartAccountClientActions,
+  type SmartAccountSigner,
+} from "@alchemy/aa-core";
+import {
+  custom,
+  type Chain,
+  type CustomTransport,
+  type HttpTransport,
+  type Transport,
+} from "viem";
+import { AlchemyProviderConfigSchema } from "../schema.js";
+import { createAlchemySmartAccountClientFromRpcClient } from "./internal/smartAccountClientFromRpc.js";
+import { createAlchemyPublicRpcClient } from "./rpcClient.js";
+import type {
+  AlchemySmartAccountClient,
+  AlchemySmartAccountClientConfig,
+} from "./smartAccountClient";
+
+// todo: this file seems somewhat duplicated with ./modularAccountClient.ts, but that file has some multi-owner specific fields. Is there a way to refactor these two to de-dupe?
```

Comment:
maybe we can create shared internal only client called like `createBaseMAClient` and have its account input be either multisig or multiowner accounts? that way the actions that are common to both are appended correctly

since both first do:

```
const client = createAlchemyPublicRpcClient({
    chain,
    connectionConfig,
  });
```

then the `account` param passed to `createBase` actually should be something in the form of `(client) => MOMA | MSMA`

I can help with the typing here if you run into issues with the generics